app/views/content/funding-and-support/scholarships-and-bursaries.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Scholarships and bursaries"
+heading: "Scholarships and bursaries"
+description: |-
+   Find out if you're eligible for extra funding depending on the subject you're training to teach.
+related_content:
+    What to expect on your teacher training: "/blog/what-to-expect-on-your-teacher-training"
+    Improve your subject knowledge: "/train-to-be-a-teacher/subject-knowledge-enhancement"
+    Salaries and benefits: "/salaries-and-benefits"
+promo_content:
+    - content/funding-and-support/promos/get-adviser-funding-promo
+navigation: 20.15
+navigation_description: Find out if you're eligible for extra funding depending on the subject you're training to teach.
+before-content:
+    - content/funding-and-support/scholarships-and-bursaries/funding-widget
```

Comment:
@ethax-ross is there a way we can move the funding widget underneath that first paragraph of content, above the ##Postgraduate bursaries and scholarships section?

Reply:
@EllieNodder I had a quick look and there's not a straight-forward way of doing this unfortunately. We have a mechanism in place to render basic components (like CTAs and quotes), but it won't work for the funding widget.

I would need to write something that lets us pull in a partial part-way down a Markdown page; its doable but I want to check if its worth the effort before I go ahead? If the way I've thought of doing this goes well its an afternoons work, but its one of those things that has the potential to be more awkward/difficult that it seems 😬

Reply:
Thanks @ethax-ross 

@EllieNodder @GDW7 My view is that this is good enough for now with the widget as it is?

@ethax-ross I'll put a ticket on the backlog to maybe explore in future - but we should see how people are using the page, whether they are scrolling down etc. to see if it is worth it